src/main/java/picard/util/help/HelpConstants.java

```diff
@@ -0,0 +1,69 @@
+package picard.util.help;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public final class HelpConstants {
+
+    private HelpConstants() {};
+
+    /**
+     * Definition of the group names / descriptions for documentation/help purposes.
+     */
+    public final static String DOC_CAT_FASTA = "Fasta File Tools";
+    public final static String DOC_CAT_FASTA_SUMMARY = "Tools for analysis and manipulation of files in fasta format";
+
+    public final static String DOC_CAT_INTERVALS = "Interval Tools";
+    public final static String DOC_CAT_INTERVALS_SUMMARY = "Tools for processing intervals and associated overlapping records";
+
+    public final static String DOC_CAT_READS = "SAM/BAM/CRAM Tools";
+    public final static String DOC_CAT_READS_SUMMARY = "Tools for manipulating read-level data (SAM/BAM/CRAM)";
+
+    public final static String DOC_CAT_VARIANT = "VCF Tools";
+    public final static String DOC_CAT_VARIANT_SUMMARY = "Tools for manipulating variants and associated metadata";
+
+    public final static String DOC_CAT_TEST = "Test Tools";
+    public final static String DOC_CAT_TEST_SUMMARY = "Tools for internal test purposes";
+
+    /**
+     * List of "supercategory" values used for doc purposes. Every doc group name can/should be put into
+     * one of the following supercategories.
+     */
+    private final static String DOC_SUPERCAT_TOOLS = "tools";
+    private final static String DOC_SUPERCAT_UTILITIES = "utilities";
+    private final static String DOC_SUPERCAT_EXCLUDE = "exclude";
+
+    private static Map<String, String> groupToSuperCategory;
+
+    private static Map<String, String> getSuperCategoryMap() {
```

Comment:
It will be nice to have this method and the supercategory values accessible to reuse this map. For example, if I create a new program group for a tool I can add it to the tools supercategory and like that reuse this class to also include the others without copying the code in this method and re-define the supercat constants....

I would like to have this also in GATK to re-use super-categories and read groups...